app/scripts/constants.js

```diff
@@ -28,6 +28,10 @@ window.OPENSHIFT_CONSTANTS = {
   CLI: {
     "Latest Release":          "https://github.com/openshift/origin/releases/latest"
   },
+  // Maps links to API UI URLs
+  API: {
+    "token-request": "https://api.preview.openshift.com/oauth/token/request"
```

Comment:
Is this the right place or should it have been thrown in the same object with the doc links above?